automobile/buildings.md

```diff
@@ -77,16 +79,16 @@ roof).
 ### SimpleBuilding PROTO
 
 The `SimpleBuilding` PROTO is similar to the `Building` one except that the wall and roof appearances don't need to be set.
-The `wallAppearance` and `roofAppearance` fields are replaced by the `wallType` and `roofType` ones.
+The `wallAppearance` and `roofAppearance` fields are replaced by the `wallType` and `roofType` ones. The `wallColor` and `roofColor` fields define respectively the color of the wall and roof. Note that not all the wall and roof types support the color argument.
```

Comment:
...fields define respectively the color of the wall and roof -> fields define the color of the wall and roof, respectively

I'd also change "Note that" to "Bear in mind that", as we tend to (over)use "Note that" elsewhere in the documentation.